alvr/server_core/src/tracking/mod.rs

```diff
@@ -373,6 +467,16 @@ pub fn tracking_loop(
                 &tracking.device_motions,
             );
 
+            if !tracking.markers.is_empty() {
+                tracking_manager_lock.report_markers(tracking.markers);
+                if matches!(
+                    headset_config.recentering_mode,
+                    RecenteringMode::Stage { .. }
+                ) {
+                    tracking_manager_lock.recenter(&headset_config.recentering_mode);
+                };
+            }
```

Comment:
We should honestly just fully make the marker case it's own function and just invoke it here.
That way we can just keep the stage case in the main recenter function as just setting the identity (which might improve behavior when switching between recentering modes, not sure). And it'd remove the not stage check from the other invocation and the weird double check that happens w/ this invocation.